superbench/common/devices/gpu.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""GPU device module."""
+
+from pathlib import Path
+
+from superbench.benchmarks import Platform
+from superbench.common.utils import logger
+
+
+class GPU():
+    """GPU device helper class."""
+    def __init__(self):
+        """Initilize."""
+        self.vendor = self.get_vendor()
+        # TODO: check CUDA or ROCm availability accordingly
+        if self.vendor == 'nvidia':
+            self.platform = Platform.CUDA
+        elif self.vendor == 'amd':
+            self.platform == Platform.ROCM
+        else:
+            self.platform = Platform.CPU
```

Comment:
`self.vendor` => `vendor`
`self.platform` => `self.__platform`
Then using @property to get platform?
```
@property
def platform(self):
    return self.__platform
```

Reply:
updated